app/assets/yge/youvegoteth/send.js

```diff
@@ -54,27 +54,12 @@ window.onload = function() {
   if (localStorage['amount']) {
     $('amount').value = localStorage['amount'];
   }
-  if (localStorage['username']) {
-    $('username').value = localStorage['username'];
-  }
-  if (localStorage['issueURL']) {
```

Comment:
Why was this taken away ? Cause `send.js` right now is used a few more places , not just tip
So I believe if we remove this  -> those might get affected (like the ETHOS flow )

Also these are just used to autopopulate the input fields , I would be a lot more comfortable if this stayed until we get around to clean up 😅 

Your thoughts @arun1595 ?

@mbeacom  / @SaptakS

Reply:
I agree. Also in try-catch, you seem to have removed some of the `setItems`

Reply:
Oh, I didn't realize this was used in other places of the app. Let me fix it.

Reply:
@thelostone-mc @SaptakS I've extracted the pre-population code to a function and it only gets called if the current page is not a send tip page. This is just a temporary fix until we clean up the dependency.

Let me know what you guys think.

/cc @mbeacom